etc/env/example.env

```diff
@@ -8,7 +8,7 @@ CANDIG_AUTH_MODULES=keycloak vault tyk opa
 
 # options are [<ip_addr>, <url>, host.docker.internal, candig.docker.internal]
 CANDIG_DOMAIN=candig.docker.internal
-CANDIG_INTERNAL_DOMAIN=candig.docker.internal
+CANDIG_INTERNAL_DOMAIN=127.0.0.1
```

Comment:
```suggestion
CANDIG_INTERNAL_DOMAIN=${CANDIG_DOMAIN}
```

Reply:
I made this change in a different PR, but we can make this change here as well.

Reply:
CANDIG_INTERNAL_DOMAIN=${CANDIG_DOMAIN} will not work since ${CANDIG_DOMAIN} is (in a server environment) set to the external URL.

UHN prod currently runs:
CANDIG_INTERNAL_DOMAIN=127.0.0.1

Reply:
Exactly...it is set specifically to a value for UHN prod, but should by default be the same as CANDIG_DOMAIN unless otherwise specified.